src/main/java/com/google/sps/servlets/AutoEventProcessor.java

```diff
@@ -0,0 +1,318 @@
+    	//Time Zones are Default RN, will update that later
+    		DateTime startDate = processDate(timeOb, "start");
+   			DateTime endDate = processDate(timeOb, "end");
+    		
+    		int startTime = startDate.getHourOfDay();
+    		int startDay = startDate.getDayOfWeek();
+
+    		int endTime = endDate.getHourOfDay();
+    		int endDay = endDate.getDayOfWeek();
+           
+    		while((startTime!= endTime) || (startDay != endDay)){
+
+    			times[startDay-1][startTime] += 1;
+    			//time starts at 0, mod operation AFTER incrementing
+    			startTime++;
+    			startTime %= 24;
+    			//Day starts at 1, mod operation BEFORE incrementing
+    		   	if(startTime == 0){
+                    startDay %= 7;
+    		   	    startDay++;
+                }
+    		}
+
+    }
+
+    private static DateTime processDate(JsonObject timeOb, String time){
+    	return new DateTime(stripQuotes(timeOb.get(time).toString())).withZone(DateTimeZone.forID("America/Los_Angeles"));
+    }
+//===============================GetFreeBusyTimes end ========================================//
+
+//===============================processFreeBusyTimes Start ========================================//
+
+
+   
+	private static String  processFreeBusyTimes(int[][] freeBusyTimes, int startConstraint, int endConstraint, int meetingLength){
+
+        	//Helper Class to Order Events
+	class Event implements Comparable<Event>{
+		public int start;
+		public int end;
+		public int day;
+        public int val;
+        public int nonAttending;
+
+		Event(int start, int end, int day, int val, int nonAttending){
+			this.start = start;
+			this.end = end;
+			this.day = day + 1;
+            this.val = val;
+            this.nonAttending = nonAttending;
+		}
+        
+        @Override
+        public int compareTo(Event o) {
+           
+            int res = this.val - o.val;
+            if (res < 0)
+                return -1;
+            else if (res > 0)
+                return 1;
+            else
+                return 0;
+        }       
+	}
+		 
+		List<Event> events = new ArrayList<Event>();
+		for(int day = 0; day < 7; day++){
+			for(int start = startConstraint; start < endConstraint - meetingLength; start++){
+                int val = freeBusyTimes[day][start] + freeBusyTimes[day][start+1] + freeBusyTimes[day][start+ 2];
+                int nonAttendees = maxNonAttending(freeBusyTimes[day][start], freeBusyTimes[day][start+1], freeBusyTimes[day][start+ 2]);
+				events.add(new Event(start, start+meetingLength, day, val, nonAttendees));
+			}
+		}
+
+		Collections.sort(events);
+        
+        for(Event e : events){
+            System.out.println(
+                "Day:   " + e.day
+             +"\nStart: " + e.start
+             +"\nEnd:   " + e.end
+             +"\nNon-attendees: " + e.nonAttending + "\n");
+        }
+        
+		return "{\"result\": true}";
+    }
+    
+    private static int maxNonAttending(int day1, int day2, int day3){
```

Comment:
Maybe adding a javadoc explaining functions and what they take and return would be nice.